galv/NeMo
# init draft for chunk offline infer and ctc ensemble

# What does this PR do ?

This is a draft.
Please don't merge!!!

TO DO:
Test the ctc implementation.
Implement the ASR clients
Test the perf
Change the naive chunk inference in nemo
    1. change the preprocessor; instead of doing inference on one sequence at one time, we should do inference of batch of sequences together.
    2. change the preprocessor and postprocessor to do chunking inference simply using the huggingface style.
